pkgs/development/python-modules/customtkinter/default.nix

```
@@ -17,20 +18,24 @@ buildPythonPackage {
   pyproject = true;
   disabled = pythonOlder "3.7";
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-/Y2zuvqWHJgu5gMNuoC0wuJYWGMHVrUTmG2xkRPY0gc=";
+  src = fetchFromGitHub {
+    owner = "TomSchimansky";
+    repo = "CustomTkinter";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-1g2wdXbUv5xNnpflFLXvU39s16kmwvuegKWd91E3qm4=";
   };
 
   nativeBuildInputs = [
     setuptools
-    wheel
+    tkinter
+  ];
+
+  propagatedBuildInputs = [
```

and for runtime ones we use dependencies

```suggestion
  dependencies = [
```